README.md

```diff
@@ -26,12 +26,12 @@ In both cases, run `mix deps.get` afterwards.
 
 There are two configuration options.
 
-One is `:routes`, which will tell the tool to generate a simple ASCII art ladder diagram. That option is a map of all the routes you want ladders for, and a list of stations along that route in order. Each station is a list of three items: the station name you want displayed on the ladder, the stop ID for `direction_id == 0` and the stop ID for `direction_id == 1` at that station.
+One is `routes`, which will tell the tool to generate a simple ASCII art ladder diagram. That option is a map of all the routes you want ladders for, and a list of stations along that route in order. Each station is a list of three items: the station name you want displayed on the ladder, the stop ID for `direction_id == 0` and the stop ID for `direction_id == 1` at that station. This will be passed into visualize as the second argument.
 
 For example:
 
 ``` ex
-config :gtfs_realtime_viz, :routes, %{
+routes = %{
   "Mattapan" => [
     ["Ashmont", "70261", "70262"],
```

Comment:
in writing this it came up that this might e better as a list of tuples - i can do that as part of this or we can do it in a future pr if you think thats worth it.

Reply:
Yeah, I initially had it as a list, because I thought the config might end up being a json file, and that doesn't have tuples. But given that we're moving the configuration to a runtime function parameter, there's no reason for that, and so I agree tuples would be better here.

Since the PR isn't _that_ big, I wouldn't mind if you just tossed it into this one. Good catch, thanks!